server/util/initCache.js

```diff
@@ -9,7 +9,7 @@ function FakeMemcached(options) {
 	const oldGet = lru.get;
 	lru.get = (key, callback) => {
 		const result = oldGet.call(lru, key);
-		return callback ? callback(result) : result;
+		return callback ? callback((result === undefined ? undefined : null), result) : result;
```

Comment:
put a switch in here so we could differentiate between "undefined" which is what the error state returns and null for no error.